### PR TITLE
Update 2.5.1 changelog.

### PR DESCRIPTION
**Pull request by rukletsov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,29 @@ Also join us on IRC in #chronos on freenode.
...
+* Push or create a PR.
+
+#### Branch off updated `stable` to `releasing-X.Y.Z`, do
+* Update `pom.xml` with the currently released version and commit.
+* Tag appropriately.
+* Push or create a PR.
+
+#### Test the release or the RC.
+* Fix code on `stable` and cherry-pick to `releasing-X.Y.Z`.
+* Update the change log on both branches.
```
Comment (gkleiman) [inline README.md]: s/change log/changelog/

File: README.md [hunk 2]
```diff
@@ -36,6 +36,29 @@ Also join us on IRC in #chronos on freenode.
...
+* Retag and retest.
+
+#### When a release is ready, send an announcement to the Chronos google group.
```
Comment (gkleiman) [inline README.md]: s/google/Google/

File: README.md [hunk 3]
```diff
@@ -36,6 +36,29 @@ Also join us on IRC in #chronos on freenode.
...
+* [if necessary] Bump other dependencies and commit.
+* Update the changelog.
+* Update `pom.xml` with a future version, major or minor, appending `-SNAPSHOT` to it.
+* Push or create a PR.
+
+#### Branch off updated `stable` to `releasing-X.Y.Z`, do
+* Update `pom.xml` with the currently released version and commit.
+* Tag appropriately.
+* Push or create a PR.
+
```
Comment (gkleiman) [inline README.md]: I'd add:

```
* Update the repository's [releases page](https://github.com/mesos/chronos/releases).
```

File: README.md (hunk 2 above)
Comment (rukletsov) [reply README.md]: not mine!

**Comment by rukletsov** [comment]
PR updated, comments addressed.

**Comment by gkleiman** [comment]
Thanks!

**Pull request merged**